code/ui/blocks/src/blocks/Description.tsx

```diff
@@ -43,58 +50,104 @@ const getInfo = (info?: Info) => info && (typeof info === 'string' ? info : str(
 
 const noDescription = (component?: Component): string | null => null;
 
-export const useDescriptionProps = (
-  { of, type, markdown, children }: DescriptionProps,
+const getDescriptionFromModuleExport = (of: DescriptionProps['of'] = 'meta'): string | null => {
```

Comment:
Would it be conventional to call this `useX`?

Reply:
that was a mistake on my part. I moved the `useOf` call out to the component to ensure it's not called conditionally. keeping this naming.